druid_setup/single/docker-compose.yml

```diff
@@ -16,8 +16,15 @@
 # specific language governing permissions and limitations
 # under the License.
 #
+name: druid_single
```

Comment:
Most of these changes have been made redundant by other recent commits - I think it's safe to drop all the changes in this file.